### ISSUE TITLE
Add "Tor-Disabled Apps" option

### ISSUE THREAD

**Issue by amirsina-mashayekh** [guardianproject/orbot#968]
Currently, there is a "Tor-Enabled Apps" option which allows us to select specific apps to pass their traffic through Tor.

However, I want all my apps to pass their traffic through Tor except a few of them. I can select all apps and then disable those, but this is not user friendly. Even there is no "Select all" option.

It would be very good if there was an option which inverted Tor-Enabled Apps; so that traffic of selected apps bypass Tor. Or at least there was a "Select All" button in VPN Mode page.

**Comment by tkbstudios** [comment]
Yeah this could be a good feature so all apps use ToR EXCEPT some apps which could be needed for some users I once had to enable each app and let 3-4 not use ToR -_- and ticking 70+ apps takes time 😂
